development/docker-compose/readme.md

```diff
@@ -183,11 +183,22 @@ To connect your application to the local environment, you can run it in the IDE
 
 The following is an example configuration:
 ```
--Deureka.client.serviceUrl.defaultZone=http://localhost:8761/eureka/
--Dbackbase.communication.http.discoverable-access-token-service=false
--Dbackbase.communication.http.access-token-uri=http://localhost:7779/oauth/token
--Dspring.activemq.broker-url=tcp://localhost:61616
--Deureka.instance.hostname=host.docker.internal
+eureka.client.enabled=true
+eureka.client.order=1
+eureka.client.serviceUrl.defaultZone=http://localhost:8761/eureka/
+eureka.instance.hostname=host.docker.internal
+spring.cloud.discovery.client.simple.order=0
```

Comment:
after making the service discoverable and registering it in the host network we still need to register the token converter to be accessible from the host network, a spring cloud hack is to configure the ordering the discovery clients this way we can have multiple discovery clients operating in simultaneous.